lesson24/src/js/main.js

```diff
@@ -0,0 +1,48 @@
+const showModal = (target)  => {
+    target.classList.remove('hidden');
+    target.setAttribute('aria-modal', 'true');
+    target.setAttribute('role', 'dialog');
+    target.removeAttribute('aria-hidden');
+}
+
+const hideModal = (target)  => {
+    target.classList.add('hidden');
+    target.setAttribute('aria-hidden', 'true');
+    target.removeAttribute('aria-modal');
+    target.removeAttribute('role');
+}
+
+const toggleModal = () => {
+    const modalTarget = document.querySelector('[data-id="popup-modal"]');
+    const modalTriggerElements = document.querySelectorAll("[data-modal-toggle]");
+    const isHiddenModalTarget = modalTarget.classList.contains("hidden");
+    
+    modalTarget.addEventListener('click',() => hideModal(modalTarget));
+    
+    modalTriggerElements.forEach((trigger) => {
+        trigger.addEventListener('click', () => {
+            isHiddenModalTarget ? showModal(modalTarget) : hideModal(modalTarget);
+        });
+    });
+}
+
+const agreeCheckBox = document.querySelector('[data-id="agree_check"]');
+const agreeButton   = document.querySelector('[data-id="agree_button"]');
+const submitButton  = document.querySelector('[data-id="submit_button"]');
+
+agreeCheckBox.addEventListener('change' , () => {
+    submitButton.disabled = agreeCheckBox.checked ? false : true;
```

Comment:
nits
```suggestion
    submitButton.disabled = !agreeCheckBox.checked
```

Reply:
Thanks for the review!
I fixed.
https://github.com/nakagawamai/js-lesson/pull/25/commits/cd7dc555bdea03cc22caef7efee33d764ce1d01e